### ISSUE TITLE
single tap doesn't draw a point

### ISSUE THREAD

**Issue by haemi** [michalkonturek/SignatureView#6]
this would be necessary for e.g. the dot on the i...

**Comment by mats019** [comment]
Yes. I am facing with the same issue. After sign if we give dot it is not accepting.
Please address.

**Comment by Lioenl-Vicky** [comment]
This issue, I found in a 4" inches size display only. The 4.7" and 5.5" are works

add the following code to a `touchesBegan()` method, at SignatureView.m

    self.image = [self _drawLineFromPoint:self.previousPoint toPoint:currentPoint image:self.image];

This works for me. ^_^
